stan/math/prim/scal/prob/neg_binomial_lccdf.hpp

```diff
@@ -122,8 +122,8 @@ namespace stan {
           ops_partials.edge1_.partials_[i] -= g1 / Pi;
         }
         if (!is_constant_struct<T_inv_scale>::value)
-          ops_partials.edge2_.partials_[i] -= d_dbl * pow(1-p_dbl, n_dbl)
-            * pow(p_dbl, alpha_dbl-1) / beta_func / Pi;
+          ops_partials.edge2_.partials_[i] -= d_dbl * pow(1 - p_dbl, n_dbl)
+            * pow(p_dbl, alpha_dbl - 1) / beta_func / Pi;
```

Comment:
`a / (b * c)` is more efficient than `a / b / c` because multiplication is faster than division.  This is true with primitive types and even more important with autodiff types.